docs/casa/manual/chapters/10_outer_frame_assembly.md

```diff
@@ -0,0 +1,151 @@
+# Outer Frame Assembly
+
+![](../img/10_1.png)
+
+The outer frame assembly of Casa, designed using 2020 V slot extrusion, is assembled in two halves. Each half is joined by a backbone of 2040 extrusion, making the enclosure easy to disassemble for moving. The panels, sliding door, electronics, and basement will attach to this main frame. Throughout this stage, remember to check for squareness and adjust accordingly.
+
+The outer frame assembly uses eight different extrusions labeled A - H. Some extrusions are the same length but feature different cross-drill patterns, so pay attention and select the correct extrusion. Extrusions A, C, D, and H are the extrusions that have cross-drilling. If you are self-sourcing, you may find the printable cross-drill guides handy.
+
+<!-- We really need a way of defining the extrusions here -->
+
+!!! tip "Blind Joints"
+    ![](../img/10_blind_joints.jpg)
+
+    Casa's frame is assembled using blind joints. These are simple and cheap joints that provide excellent strength through metal-to-metal contact. Blind joints require cross-drilling the extrusion to allow access to tighten down a button head bolt and also require manual alignment. Make sure that joints are aligned and square when tightening.
+
+    **All blind joints are secured using an M5 x 10 BHCS screw.** In the interest of simplifying this manual, we have chosen to omit the button head bolts. Wherever a part interfaces with extrusions in a way that looks like it requires a bolt for a blind joint, then it should be considered a part that requires a bolt.
+
+---
+
+!!! info annotate "Components Required"
+    ```
+        1 x Extrusion A
+        3 x Extrusion B
+        3 x M5 x 10 BHCS
+    ```
+![](../img/10_s1.png)
+
+---
+
+!!! info annotate "Components Required"
+    ```
+        1 x Extrusion A
+        3 x M5 x 10 BHCS
+    ```
+![](../img/10_s2.png)
+
+!!! note "Orientation"
+    Note that this part is **not** symmetrical. The central cross beam sits closer to the bottom of the enclosure than to the top.
```

Comment:
Thoughts on adding some reference dimensions to the image to show that it's asymmetrical?

Reply:
I will give it some thought